apps/aehttpserver/lib/aehttpserver/endpoint.ex

```diff
@@ -35,7 +35,7 @@ defmodule Aehttpserver.Endpoint do
 
   plug Aehttpserver.Router
 
-  def on_response(status, headers, _body, request) do
-    {status, List.keyreplace(headers, "server", 0, {"server", "aehttpserver"}), request}
-  end
+  # def on_response(status, headers, _body, request) do
```

Comment:
Why is this code getting commented out instead of being removed?
If the code is being commented out temporarily, you should leave a TODO explaining what is required to add it back in.
If the code is not needed, it should be removed entirely

Reply:
correct, thank you